src/modules/remove_task.js

```diff
@@ -0,0 +1,22 @@
+import toDoList from './toDoList_data.js';
+import display from './display.js';
+import updateLocalStorage from './update_local_sorage.js';
+
+function removeTask(event) {
+  const indexToRemove = event.target.parentNode.value;
+  toDoList.splice(indexToRemove, 1);
+  let newIndex = 0;
+  toDoList.forEach((task) => {
+    task.index = newIndex;
+    newIndex += 1;
+  });
+  display(toDoList);
```

Comment:
- [ ]  You have done an amazing job up to this point. However,I would suggest that start indexing your tasks from 1 and not 0 to meet this requirement:  

![image](https://github.com/Diegogagan2587/To-Do-List/assets/98453979/5be2be70-b7b0-46f1-81c2-52ee85526639)

Reply:
@frank1738 I do appreciate the feedback, Thanks for taking the time, will do it!